app/models/ems_refresh.rb

```diff
@@ -180,7 +180,7 @@ def self.queue_merge(targets, ems, create_task = false)
       task_id = if msg && msg.task_id
                   msg.task_id.to_i
                 elsif create_task
-                  task = create_refresh_task(ems, targets)
+                  task = create_refresh_task(ems, targets) # This use case is hard/ugly to fix
```

Comment:
@Fryguy This case is hard. Will need to adapt `queue_merge(:create_task => true)` to use the new `generic_action_with_callback`. But then we'll have 2 very different ways 😢